app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Expr.scala

```diff
@@ -45,6 +45,7 @@ final case class EeittCtx(value: Eeitt) extends Expr
 final case class UserCtx(value: UserField) extends Expr
 final case class Constant(value: String) extends Expr
 final case object Value extends Expr
+final case object SubmissionReference extends Expr
```

Comment:
Extend ExprGen to include this case so that it's included in the JSON round tripping tests.